apps/web/src/components/Publication/OpenAction/UnknownModule/Decent/CurrencySelector.tsx

```diff
@@ -0,0 +1,215 @@
+import type { AllowedToken } from '@hey/types/hey';
+import type { FC } from 'react';
+import type { Address } from 'viem';
+
+import { STATIC_IMAGES_URL } from '@hey/data/constants';
+import { TokenContracts } from '@hey/data/contracts';
+import getRedstonePrice from '@hey/helpers/getRedstonePrice';
```

Comment:
We no longer need this, `allowedTokens` list already have USD value in it

Reply:
`allowedToken` doesn't seem to contain the USD value
<img width="477" alt="image" src="https://github.com/heyxyz/hey/assets/18723363/cbab6dc4-5fc4-4c30-9d91-9c18534f8e7e">

Reply:
Let's use the USDC value 🙇🏼